### ISSUE TITLE
Documentation outline font too small. Is it possible to customize the styles of outline?

### ISSUE THREAD

**Issue by dracodoc** [rstudio/rstudio#6887]
I use RMarkdown extensively, and with long documents the outline is very important to navigate between chapters, functions, and chunks. However the outline font is way to small and hard to read.

I can adjust the font with inspector easily, but the outline css class is dynamically generated so I cannot make a permanent fix.

If only the outline pane have a fixed css class, we will be able to adjust the styles easily.

Besides, it will be even better if we can fold on certain level per document. For example we may want to fold to chunk level so it will not show levels below chunk, this way the outline will be much shorter. Right now I have too many functions and this make navigating quite hard.

**Comment by R-Hannibal** [comment]
Thanks for the suggestion @dracodoc . We'll have our development team look at it and see if we can work it into a future release.

**Comment by scisus** [comment]
I really wish this issue hadn't been closed - this is an accessibility issue.

**Comment by ronblum** [comment]
@scisus Thanks for noting this! I'll reopen in order to review for the reason you raise.

**Comment by gtritchie** [comment]
Verified on desktop build 2024.10.0-191 on macOS, and on Server 2024.10.0-192 on Ubuntu 24. Checked both source mode and visual mode outlines and both are obeying the new preference.
